data/lib/core/highscores.lua

```diff
@@ -0,0 +1,242 @@
+function Vocation.getRelated(self)
```

Comment:
Any reason not to move to vacation.lua?

Reply:
It needs to work more before it goes on vacation

Reply:
(it's because highscores get loaded before vocations. we really need to start using `require`)

Reply:
What if I make a global function `setUpHighscores` to initialize those arrays (`clientIdsVocation` and `relatedIdsVocation`), call the function on the startup global event and move `Vocation.getRelated` and `Vocation.getFromVocation` to `vocation.lua`? @MillhioreBT